### ISSUE TITLE
issue adding user to FUID Database

### ISSUE THREAD

**Issue by Simperdy** [Forcepoint/fp-bd-fuid-cisco-pxgrid#1]
Hey,

I just started a setup as mentioned in your guide https://forcepoint.github.io/docs/fuid_cisco_ise_pxgid/

Server 1: Forcepoint User Identity Service v2.3 (maybe the fuid version is too new?)
- LDAP connection enabled --> working (imported users and groups)

Server 2: fp-bd-fuid-cisco-pxgrid Service
- pxGrid connection --> working
- LDAP connection --> working
- FUID connection --> working until adding LDAP connection to fuid service on server 1

Server 1 fuid service journalctl log
```
Aug 19 16:36:52 servername fuid[4214]: time="2021-08-19T16:36:52+02:00" level=error msg=Request errors="postgres error: integrity_constraint_violation (23505)" host="servername.domain.com:5000" latency=8.922726ms method=POST path="/api/uid/v1.0/user/13df7877-c6c9-4233-a41e-8068cb5a0ce4" remote_ip=x.x.x.x status=409
```

Server 2 pxGrid service log
```
time="2021-08-19T16:24:31+02:00" level=info msg="DOMAIN\\USERNAME"
time="2021-08-19T16:24:32+02:00" level=warning msg="User 'USERNAME' is not exist in FUID Database"
time="2021-08-19T16:24:32+02:00" level=info msg="Connecting with AD Domain Conttroler ldaps.domain.com"
time="2021-08-19T16:24:32+02:00" level=info msg="Read User Object from AD Domanin Controller for user USERNAME"
time="2021-08-19T16:24:32+02:00" level=error msg="add user to FUID statusCode 409 409 Conflict"
```

Any idea how to solve this issue without going back to fuid service v2.0?

**Comment by Simperdy** [comment]
Hey guys, any updates on this?

**Comment by Simperdy** [comment]
I got additional information.
Seems like FUID Service 2.3 uses the wrong ntlm-identity.
The FUID Service syncs the data with the wrong ntlm-identity to the database.
The pxgrid service now tries to query the user with the correct ntlm-identity and gets a 404 (not found) from the api.
The pxgrid service then tries to create the user via api but gets a 409 error because the entry is not unique.

I opened a case and wait for a reply of the support.
